### PR TITLE
Add support for Archetype Minerva LX

### PR DESCRIPTION
**Pull request by archetype-made**
## Description
First support for Archetype Minerva

## QMK Pull Request 
https://github.com/qmk/qmk_firmware/pull/23291

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

File: v3/archetype/minervalx/Minerva_LX_via_040424.json [hunk 1]
```diff
@@ -0,0 +1,248 @@
+{
+  "name": "Minerva LX",
+  "vendorId": "0x5453",
+  "productId": "0x0100",
+  "keycodes": ["qmk_lighting"],
+  "menus": ["qmk_backlight"],
+  "matrix": { "rows": 5, "cols": 15 },
```
Comment (dunk2k) [inline v3/archetype/minervalx/Minerva_LX_via_040424.json:7]:
```suggestion
  "keycodes": [
    "qmk_lighting"
  ],
  "menus": [
    "qmk_backlight"
  ],
  "matrix": {
    "rows": 5,
    "cols": 15
  },
```

File: v3/archetype/minervalx/Minerva_LX_via_040424.json [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+    "labels": [
+      "Split Backspace",
+      "Split Left Shift",
+      [
+          "Bottom Row",
+          "Tsangan",
+          "Tsangan-Split",
+          "ANSI"
+      ]
+  ],
```
Comment (dunk2k) [inline v3/archetype/minervalx/Minerva_LX_via_040424.json:18]:
```suggestion
    ],
```

File: v3/archetype/minervalx/Minerva_LX_via_040424.json [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+  "matrix": { "rows": 5, "cols": 15 },
+  "layouts": {
+    "labels": [
+      "Split Backspace",
+      "Split Left Shift",
+      [
+          "Bottom Row",
+          "Tsangan",
+          "Tsangan-Split",
+          "ANSI"
```
Comment (dunk2k) [inline v3/archetype/minervalx/Minerva_LX_via_040424.json:16]:
```suggestion
        "Bottom Row",
        "Tsangan",
        "Tsangan-Split",
        "ANSI"
```

**Review by dunk2k — CHANGES_REQUESTED** [review]
formatting

**Review by Cipulot — CHANGES_REQUESTED** [review]

**Comment by Cipulot** [comment]
@archetype-made please change the file name so that it's `v3/archetype/minervalx/minervalx.json` instead of `v3/archetype/minervalx/Minerva_LX_via_040424.json`.

**Review by Cipulot — APPROVED** [review]

File: v3/archetype/minervalx/minervalx.json [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+        },
+        "4,12\n\n\n2,2",
+        {
+          "w": 1.25
+        },
+        "4,13\n\n\n2,2"
+      ]
+    ]
+  }
+}
```
Comment (Cipulot) [inline v3/archetype/minervalx/minervalx.json:248]: - formatting
- fix of color code for some keys
- fix of overlapping layout options that make it impossible to see what is actually there

```suggestion
{
  "name": "Minerva LX",
  "vendorId": "0x5453",
  "productId": "0x0100",
  "matrix": {
    "rows": 5,
    "cols": 15
  },
  "keycodes": [
    "qmk_lighting"
  ],
  "menus": [
    "qmk_backlight"
  ],
  "layouts": {
    "labels": [
      "Split Backspace",
      "Split Right Shift",
      [
        "Bottom Row",
        "Tsangan",
        "Tsangan-Split",
        "ANSI"
      ]
    ],
    "keymap": [
      [
        {
          "c": "#777777"
        },
        "0,0",
        {
          "c": "#cccccc"
        },
        "0,1",
        "0,2",
        "0,3",
        "0,4",
        "0,5",
        "0,6",
        "0,7",
        "0,8",
        "0,9",
        "0,10",
        "0,11",
        "0,12",
        {
          "c": "#aaaaaa",
          "w": 2
        },
        "0,13\n\n\n0,0",
        {
          "x": 0.25,
          "c": "#cccccc"
        },
        "0,14",
        {
          "x": 0.5
        },
        "2,12\n\n\n0,1",
        {
          "c": "#aaaaaa"
        },
        "0,13\n\n\n0,1"
      ],
      [
        {
          "w": 1.5
        },
        "1,0",
        {
          "c": "#cccccc"
        },
        "1,1",
        "1,2",
        "1,3",
        "1,4",
        "1,5",
        "1,6",
        "1,7",
        "1,8",
        "1,9",
        "1,10",
        "1,11",
        "1,12",
        {
          "w": 1.5
        },
        "1,13",
        {
          "x": 0.25
        },
        "1,14"
      ],
      [
        {
          "c": "#aaaaaa",
          "w": 1.75
        },
        "2,0",
        {
          "c": "#cccccc"
        },
        "2,1",
        "2,2",
        "2,3",
        "2,4",
        "2,5",
        "2,6",
        "2,7",
        "2,8",
        "2,9",
        "2,10",
        "2,11",
        {
          "c": "#777777",
          "w": 2.25
        },
        "2,13",
        {
          "x": 0.25,
          "c": "#cccccc"
        },
        "2,14"
      ],
      [
        {
          "c": "#aaaaaa",
          "w": 2.25
        },
        "3,0",
        {
          "c": "#cccccc"
        },
        "3,1",
        "3,2",
        "3,3",
        "3,4",
        "3,5",
        "3,6",
        "3,7",
        "3,8",
        "3,9",
        "3,10",
        {
          "c": "#aaaaaa",
          "w": 1.75
        },
        "3,12\n\n\n1,1",
        {
          "x": -1.75,
          "w": 2.75
        },
        "3,12\n\n\n1,0",
        {
          "x": -1
        },
        "3,13\n\n\n1,1",
        {
          "x": 0.25,
          "c": "#cccccc"
        },
        "3,14"
      ],
      [
        {
          "c": "#aaaaaa",
          "w": 1.5
        },
        "4,0\n\n\n2,0",
        "4,1\n\n\n2,0",
        {
          "w": 1.5
        },
        "4,2\n\n\n2,0",
        {
          "c": "#cccccc",
          "w": 7
        },
        "4,6\n\n\n2,0",
        {
          "c": "#aaaaaa",
          "w": 1.5
        },
        "4,10\n\n\n2,0",
        "4,12\n\n\n2,0",
        {
          "w": 1.5
        },
        "4,13\n\n\n2,0",
        {
          "x": 0.25
        },
        "4,14"
      ],
      [
        {
          "y": 0.25,
          "w": 1.5
        },
        "4,0\n\n\n2,1",
        "4,1\n\n\n2,1",
        {
          "w": 1.5
        },
        "4,2\n\n\n2,1",
        {
          "c": "#cccccc",
          "w": 3
        },
        "4,4\n\n\n2,1",
        {
          "c": "#aaaaaa"
        },
        "4,6\n\n\n2,1",
        {
          "c": "#cccccc",
          "w": 3
        },
        "4,8\n\n\n2,1",
        {
          "c": "#aaaaaa",
          "w": 1.5
        },
        "4,10\n\n\n2,1",
        "4,12\n\n\n2,1",
        {
          "w": 1.5
        },
        "4,13\n\n\n2,1"
      ],
      [
        {
          "w": 1.25
        },
        "4,0\n\n\n2,2",
        {
          "w": 1.25
        },
        "4,1\n\n\n2,2",
        {
          "w": 1.25
        },
        "4,2\n\n\n2,2",
        {
          "c": "#cccccc",
          "w": 6.25
        },
        "4,6\n\n\n2,2",
        {
          "c": "#aaaaaa",
          "w": 1.25
        },
        "4,9\n\n\n2,2",
        {
          "w": 1.25
        },
        "4,10\n\n\n2,2",
        {
          "w": 1.25
        },
        "4,12\n\n\n2,2",
        {
          "w": 1.25
        },
        "4,13\n\n\n2,2"
      ]
    ]
  }
}

```

**Review by Cipulot — CHANGES_REQUESTED** [review]

**Review by Cipulot — APPROVED** [review]

**Pull request merged**